### ISSUE TITLE
Feature request - apply hysteresis only to temp drops, not increases

### ISSUE THREAD

**Issue by aufkrawall** [Rem0o/FanControl.Releases#34]
Hello,
thanks for this fantastic program.

I'd like to have an option that applies hysteresis only to drops in temperature, not increases. This would help to avoid constant changes of fan speed while the cooled component doesn't get hotter than necessary. Perhaps simply negative values could be added to offered the hysteresis values of a graph?

Thanks!

**Comment by C0NTi** [comment]
I too would find something like this useful, 
I would like to be super responsive to temperature increases but not so much to temp drops, to avoid the constantly fluctuating fan speeds.

Maybe segregated +ve and -ve values.

**Comment by Rem0o** [comment]
Went over the subject, decided to make the switch in V50. It now only applies on the way down. Response time does still apply though. 

https://github.com/Rem0o/FanControl.Releases/releases/tag/V50

**Comment by aufkrawall** [comment]
Thank you. Though I wonder if it works as intended, as I set hysteresis to 6°C for my GPU graph, but as soon as temperature of the GPU core sensor reports a dip by from 70°C to 69°C, also fan speed decreases.

**Comment by Rem0o** [comment]
@aufkrawall there might be an issue with the default value when you create a new curve. If you set any value to the Hysteresis it should work properly.

**Comment by aufkrawall** [comment]
I changed hysteresis from 6°C to 3°C and it still seems to get ignored. :(

**Comment by aufkrawall** [comment]
@Rem0o I think it generally works now with v53 (I've also started with a fresh profile). Though I got the suspicion that the hysteresis stops at 65°C when my graph starts to be flat below it:
![curve](https://user-images.githubusercontent.com/13510490/97121395-358fb400-171e-11eb-9f47-620e733920c6.png)

So e.g. temperature is 69°C, fans run at 1100rpm accordingly and hysteresis is set to 9°C. Shouldn't the fans start slowing down only once 60°C is reached? It does so already when reaching 65°C.
